db/migrate/20160615013423_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
I think in terms of future scalability, we should include it in our initial design. What do you think Andrew?